check_copyright.py

```diff
@@ -508,7 +508,10 @@ def check_copyrights(args: argparse.Namespace, config: configparser.ConfigParser
             continue
 
         # Is this file a new file
-        is_new_file = args.is_new_file[file_name]
```

Comment:
I believe simple `is_new_file = args.is_new_file.get(file_name)` would do the same trick.

Reply:
Indeed!  I `C` 😂 
I have updated the PR. PTAL. Thanks!

Reply:
Dennis Ritchie is the best of course :)